src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -926,17 +926,23 @@ private static void VerifyModelForOutVarInNotExecutableCode(SemanticModel model,
             }
         }
 
-        private static void AssertInfoForDeclarationExpressionSyntax(SemanticModel model, DeclarationExpressionSyntax decl)
+        private static void AssertInfoForDeclarationExpressionSyntax(SemanticModel model, DeclarationExpressionSyntax decl, Symbol expectedSymbol = null, TypeSymbol expectedType = null)
         {
             var symbolInfo = model.GetSymbolInfo(decl);
-            Assert.Null(symbolInfo.Symbol);
+            if (expectedSymbol != null)
```

Comment:
> if (expectedSymbol != null) [](start = 11, length = 28)

Please remove the 'if', we should assert that we get null when null is expected.